cli/src/helper.rs

```diff
@@ -51,3 +55,41 @@ pub async fn bytes_from_hex_or_file(hex: Option<String>, file: Option<String>) -
         None
     })
 }
+
+pub async fn get_mnemonic() -> Result<String, Error> {
+    let mnemonic = generate_mnemonic()?;
+    println_log_info!("Mnemonic has been generated.");
+
+    let choices = [
+        "Write To Console Only",
+        "Write To File Only",
+        "Write To Console and File",
```

Comment:
What's up with the weird capitalisation?

Reply:
I'd remove them, feels inconsistent

Reply:
It's actually called `https://en.wikipedia.org/wiki/Title_case`, but I messed it up a bit.

Reply:
Let's just do a normal sentence anyway